### DISCUSSION TITLE
jdownloader container not installing curl

### DISCUSSION THREAD

**Discussion by jcmbowman** [jlesage/docker-jdownloader-2#186] [Q&A] [answered]
For some reason after my jdownloader docker container is built it doesn't contain the curl binary. This is odd, because I see the install command right there at line 26 in the dockerfile. 

If I launch a shell inside container and run a "which curl" it finds nothing. I can add it using the same command that's in the dockerfile - "apk --no-cache add curl" and then everything is fine, but it seems odd to me that it wouldn't already be there.

Any ideas? I'm trying to use curl as part of a healthcheck command.

**Comment by jlesage** [comment, marked as answer]
`curl` is used to build the image, but it is not part of the image itself.

However, you can use the `INSTALL_PACKAGES` environment variable to install it automatically.  See https://github.com/jlesage/docker-jdownloader-2#environment-variables

**Reply by jcmbowman** [reply]
Thank you so much - this is exactly the clarification I was looking for.
